### ISSUE TITLE
Unable to compile latest version for Windows

### ISSUE THREAD

**Issue by Kryptoxic** [EliasOenal/multimon-ng#149]
I tried to compile with both MingW64 and VS2019. VS2019 just gives me a "Sucessfully built target multimon-ng.exe" message but the .exe is nowhere to be found. Same with MingW64, it shows that it sucessfully compiled but the .exe is nowhere to be found at all. Any solutions as to how to compile for Windows?

**Comment by eabase** [comment]
Yeah, just had the same problem. Must be an issue in one of the CMake files?

**Comment by erikklavora** [comment]
@eabase did you solve this problem? it would be nice to have the compiled file on here :)
